pkg/client/kubernetes/tunnel.go

```diff
@@ -59,7 +59,6 @@ func NewTunnel(client rest.Interface, config *rest.Config, namespace, podName st
 // Close disconnects a tunnel connection
 func (t *Tunnel) Close() {
 	close(t.stopChan)
-	close(t.readyChan)
```

Comment:
Is this correct? That channel get's close nowhere now...

Reply:
thats correct. While debugging I got multiple panics  because of a double close at this point. The tunnel.go method is copied from helm and that had this changed as well: https://github.com/kubernetes/helm/commit/fe3eeaf39d44a95aa9e7cff29647cd12ab60da5a.
Its totally unrelated but not worth a separate PR I think.

Reply:
Ok, sure thing